.github/workflows/build_and_test.yml

```diff
@@ -11,12 +11,13 @@ on:
 
 jobs:
   build_and_test_master:
-    name: Build and Test Master ROS ${{ matrix.ros_version }} ${{ matrix.ros_distro }}
+    name: Build and Test ROS ${{ matrix.ros_version }} ${{ matrix.ros_distro }}
```

Comment:
I used the build type but like branch name better.

Reply:
My mistake: note how we don't even care about the branch because the repos file is used. The schedule action runs for the default branch (same for the checkout action if not triggered by the non-default branch).